pandas/tests/test_strings.py

```diff
@@ -2568,6 +2568,16 @@ def test_get(self):
         expected = Series(['3', '8', np.nan])
         tm.assert_series_equal(result, expected)
 
+        # complex structures
+        values = Series([(1, 2, 3), [1, 2, 3], {1, 2, 3},
+                         {1: 'a', 2: 'b', 3: 'c'}])
+
+        result = values.str.get(1)
+        expected = Series([2, 2, np.nan, 'a'])
+
+        result = values.str.get(-1)
+        expected = Series([3, 3, np.nan, np.nan])
```

Comment:
can you add .get for embedded lists/tuples/ndarrays (you may want to parameterize)

Reply:
Not 100% if by embedded you meant nested, let me know if what I implemented is not what you meant.